backend/routes/performance_routes.py

```diff
@@ -76,7 +76,8 @@ def get_adjudications_by_performance(performance_ids):
 
 @blueprint.route('/<int:performance_id>/adjudications', methods=['GET'])
 def get_adjudications(performance_id):
-    adjudications = Adjudication.get_by(performance_id=performance_id)
+    # Query by performance id and any additional query parameters provided
+    adjudications = Adjudication.get_by(performance_id=performance_id, **request.args.to_dict())
```

Comment:
Just making sure you tested this -- I think this works but not entirely sure LOL

Reply:
Also, can we put a name on `request.args.to_dict()` for readability